Hi @a-kenji, looks like it's also failing on my side for the same reason.

The service failed because: `paperless-web.service: State 'stop-sigterm' timed out. Killing.`

Then,

```log
postgres: waiting for unit paperless-web.service
!!! Test "Exporter" failed with error: "unit "paperless-web.service" reached state "failed""
!!! Traceback (most recent call last):
!!!   File "<string>", line 76, in <module>
!!!     test_paperless(postgres)
!!!   File "<string>", line 56, in test_paperless
!!!     node.wait_for_unit("paperless-web.service")
!!! 
!!! RequestedAssertionFailed: unit "paperless-web.service" reached state "failed"
```

--- a/nixos/modules/services/misc/paperless.nix
+++ b/nixos/modules/services/misc/paperless.nix
@@ -23,6 +23,7 @@ let
       PAPERLESS_THUMBNAIL_FONT_NAME = defaultFont;
       GRANIAN_HOST = cfg.address;
       GRANIAN_PORT = toString cfg.port;
+      GRANIAN_WORKERS_KILL_TIMEOUT = "60";
     }
     // lib.optionalAttrs (config.time.timeZone != null) {
       PAPERLESS_TIME_ZONE = config.time.timeZone;

